crates/codegen/parser/runtime/src/templates/kinds.rs.jinja2

```diff
@@ -13,6 +13,7 @@ use napi_derive::napi;
     strum_macros::Display,
     strum_macros::EnumString,
 )]
+#[strum(use_phf)]
```

Comment:
`phf` crate looks like a great addition! but I wonder if it is required for the query engine.
If not, what do you think about adding it separately, possibly once we have a perf suite, so that we can measure/report a realistic delta (for this use case, and possibly others)?

Reply:
I originally put this in because I was doing the string -> enum conversion in an inner loop, but then I took that out, and so this wasn't needed. But I decided to leave it in because that operation is likely to be used by clients, and they have no way of doing this optimisation themselves because it needs to be applied here. Plus I think that phf should be the default way this is implemented.

The cost is small, and even though for *our* current code it is an unneccesary optimisation, unjustified by profiling, we cannot say the same for client code, and only we can make this choice.

Reply:
> But I decided to leave it in because that operation is likely to be used by clients, and they have no way of doing this optimisation themselves because it needs to be applied here. Plus I think that phf should be the default way this is implemented.

fully agree .. but if it is unrelated to/required by the query engine, I suggest doing it separately.

And I agree that it is going to be an improvement. But if we have a perf suite, it is an excellent use case/test case to use for it. Not to justify it, but to prove how much we care and affect performance by the work we are currently doing.